frontend/src/main/pages/HomePage.css

```diff
@@ -0,0 +1,11 @@
+.gradient-animation {
```

Comment:
There is a better place to put CSS, but we'll leave cleaning that up to another PR.